jesterTOV/inference/flows/models/nicer_maf/J00300451/retrain_amsterdam_flows.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Submit all Amsterdam J0030 flow training jobs to the cluster.
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+for dir in "$SCRIPT_DIR"/amsterdam_*/; do
+    echo "Submitting: $dir"
+    cd "$dir"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)